jdat_notebooks/ifu_optimal/requirements.txt

```diff
@@ -0,0 +1,14 @@
+notebook==6.1.4
+jupyter_client==5.3.5
+numpy==1.19.2
+scipy==1.5.2
+specutils==1.1.1
+glue-astronomy==0.1
+glue-core==1.0.0
+glue-jupyter==0.2.1
+glue-vispy-viewers==1.0.1
+jdaviz==1.0.3
+regions==0.4
+photutils==1.0.1
+astropy==4.2
+matplotlib==3.3.2
```

Comment:
This looks good to me, the only thing we need to add to the end of the requirements file is:
```
ipyvuetify==1.6.2
ipyvue==1.5.0
```
This is because the auto resolve is not working for `jdaviz`

Reply:
OK, added these requirements